spec.md

```diff
@@ -546,11 +553,19 @@ Some platforms MAY support proxying of application requests to service instances
 
 The `requires` field in the [Catalog](#catalog-mgmt) endpoint enables a platform marketplace to validate requests to create bindings. A platform MAY reject requests to create bindings when a broker has declared `"requires":["route_forwarding"]` for a service in the catalog endpoint.
 
+Brokers MUST NOT include a `route_service_url` in a create binding response
+if it did not include a `"requires":["route_forwarding"]` property in its
+[Catalog](#catalog-mgmt) endpoint.
+
```

Comment:
yea, I think I got myself confused when I tried to keep these two (very related) PR separate. I'm going to merge them into one.